clusterloader2/pkg/test/test.go

```diff
@@ -35,7 +35,7 @@ var (
 )
 
 // RunTest runs test based on provided test configuration.
-func RunTest(f *framework.Framework, conf *api.Config) []error {
+func RunTest(basepath string, f *framework.Framework, conf *api.Config) []error {
```

Comment:
Could you move this `basepath` parameter to framework instead of passing it as separate argument in so many functions? I think it also makes sense to have it in there.

Reply:
Good idea. Done.